src/utilities/theme/utils.ts

```diff
@@ -41,10 +42,16 @@ export function buildThemeContext(
   themeConfig: ThemeConfig,
   cssCustomProperties?: CustomPropertiesLike,
 ): Theme {
-  const {logo} = themeConfig;
+  const {
+    logo,
+    UNSTABLE_colors,
+    colorScheme = DEFAULT_COLOR_SCHEME,
```

Comment:
It feels like the normalisation to set a default value for the colorScheme should have already happened by this point. It feels like that should happen at the same point we turn custom properties object, that we pass into this function, rather than trying to work it out in here.